spec/models/card_spec.rb

```diff
@@ -42,6 +42,14 @@
           expect(card.incorrect_answers).to eq 1
         end
       end
+
+      context 'when user made typo in word' do
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.